client/src/components/PhotoGallery.jsx

```diff
@@ -1,27 +1,72 @@
 import React from 'react';
 import styled from 'styled-components';
+import PhotoGalleryModal from './PhotoGalleryModal.jsx';
 
 const StyledPhotoGallery = styled.div`
   grid-row: 2;
   grid-column: 6 / -1;
-  border-style: solid;
+
+  margin-bottom: 1100px;
 `;
 
-class PhotoGallery extends React.Component {
-  constructor(props) {
-    super(props);
+const StyledSeeAllPhotos = styled.span`
+  color: #0E70BE;
+  :hover {
+    cursor: pointer;
+  }
+  text-decoration: underline;
+  font-size: 12px;
+  float: right;
+  grid-row: 1;
+  grid-column: 1;
+  text-align: right;
+`;
+
+const StyledCustomerPhotosTitle = styled.span`
+  margin-bottom: 0px;
+  margin-top: 32px;
+  font-weight: bold;
+  font-size: 14px;
+  grid-row: 1;
+  grid-column: 1;
+`;
 
-    // this.state = {
+const StyledPhotoGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
```

Comment:
Ah, so THAT's how you do this! I've been trying to do grid-template-columns: (4, 1fr); with no success, but I see how I forgot the repeat keyword. Nice work.